### DISCUSSION TITLE
Missing Function in Openzepplin Package

### DISCUSSION THREAD

**Discussion by SpringxDSay** [Cyfrin/foundry-full-course-cu#2374] [Q&A] [answered]
while trying to flip the mood of the NFT is MoodNft.sol contract,  to ascertain that the only the owner can flip the mood, _isApprovedOrOwner() function in the openzepplin package was used but it couldn't be found. Is there now an alternate function to use instead??

**Comment by SpringxDSay** [comment, marked as answer]
I've seen that getApproved() and ownerOf() were used instead 👍🏽

**Reply by EngrPips** [reply]
Interesting. Keep crushing it.

**Reply by SpringxDSay** [reply]
ty buddy

**Reply by EngrPips** [reply]
Anytime.
